### PR TITLE
Build with MQTT and Modbus Support in Github Workflow 

### PR DESCRIPTION
**Pull request by mhansal**

### CODE REVIEW

File: .github/workflows/cmake-multi-platform.yml [hunk 1]
```diff
@@ -79,6 +79,8 @@ jobs:
         -DFORTE_COM_FBDK=ON
         -DFORTE_COM_HTTP=ON
         -DFORTE_COM_LOCAL=ON
+        -DFORTE_COM_MODBUS=${{ matrix.os == 'windows-latest' && 'OFF' || 'ON' }}
+        -DFORTE_COM_PAHOMQTT=${{ matrix.os == 'windows-latest' && 'OFF' || 'ON' }}
```
Comment (MartinMelikMerkumians) [inline .github/workflows/cmake-multi-platform.yml]: Wouldn't it be nicer to put the value into a variable and set it in the matrix?

File: .github/workflows/cmake-multi-platform.yml [hunk 2]
```diff
@@ -57,9 +57,9 @@ jobs:
       run: |
         echo "build-output-dir=${{ github.workspace }}/build" >> "$GITHUB_OUTPUT"
     
-    - name: Install libboost (Linux)
+    - name: Install libboost, libmodbus and libpaho-mqtt (Linux)
       if: ${{ matrix.os != 'windows-latest' }}
-      run: sudo apt-get install -y libboost-all-dev
+      run: sudo apt-get install -y libboost-all-dev libmodbus-dev libpaho-mqtt-dev
```
Comment (MartinMelikMerkumians) [inline .github/workflows/cmake-multi-platform.yml]: May I ask for the reason to not build paho under windows?

Comment (mhansal) [reply .github/workflows/cmake-multi-platform.yml]: Fixed.

File: .github/workflows/cmake-multi-platform.yml (hunk 1 above)
Comment (mhansal) [reply .github/workflows/cmake-multi-platform.yml]: Fixed.

**Comment by cochicde** [comment]
closing as per https://github.com/eclipse-4diac/4diac-forte/pull/228#issuecomment-2452165133

**Pull request closed without merging**